src/main/java/org/isf/patient/model/PatientProfilePhoto.java

```diff
@@ -53,17 +55,17 @@ public class PatientProfilePhoto implements Serializable {
 
 	@Column(name="PAT_PHOTO")
 	@Lob
-	private byte[] photo;
+	private Byte[] photo;
```

Comment:
I don't understand the desire to user objects when the corresponding primitives exist.   What am I missing?  Why is this done?

Reply:
I needed it for the converter, for automatization. The ModelMapper library accepts generic objects.

Reply:
Please take a look to the PatientMapping class

Reply:
Primitive vs Object is a topic: it affects performances (I don't know how much though) but also it depends if we need to set a field as `null` rather its default value. I think that there is not a one-for-all rule, but in case we have some fields that might be "not set", we should use Object types, otherwise primitives are enough.

In this case, the ModelMapper implementation requires it, otherwise a manual boxing/unboxing needs to be done anyway.

Am I wrong?

Reply:
Ok, I'll restore the previous type

Reply:
done

Reply:
with the new mapper something goes wrong on patient's picture read/write from "ui" (serialized picture)

Reply:
with latest changes now we have byte[] (primitive) and patient's picure read/write working from "ui".
thanks"!